### PR TITLE
Translate guides/release/tutorial/part-1/index.md

### LINKED ISSUE #139: Translate `/tutorial/part-1/index.md`, Ember 4.7
Please assign yourself to the issue or let a comment at the very moment you start the translation.<br><br>File: `guides/release/tutorial/part-1/index.md`<br>From Ember: **N/A**<br>To: **4.7**

### PR DESCRIPTION
**Pull request by BlueCutOfficial**
Closes #139

### CODE REVIEW

File: guides/release/tutorial/part-1/index.md [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
+<img src="/images/tutorial/part-2/provider-components/homepage-with-rentals-component@2x.png" alt="L'app Super Rentals terminée" width="1024" height="1327">
 
-<img src="/images/tutorial/part-2/provider-components/homepage-with-rentals-component@2x.png" alt="The finished Super Rentals app" width="1024" height="1327">
+<!-- spell ignore -->
+En cours de route, vous apprendrez tout ce qu'il y a à savoir pour construire une application Ember basique. Si vous vous retrouvez bloqué à n'importe quel passage du tutoriel, n'hésitez pas à télécharger <https://github.com/ember-learn/super-rentals/tree/super-rentals-tutorial-output> pour voir le code fonctionnel complet.
 
-Along the way, you will learn everything you need to know to build a basic Ember application. If you get stuck at any point during the tutorial, feel free to download <https://github.com/ember-learn/super-rentals/tree/super-rentals-tutorial-output> for a complete working example.
+Ce tutoriel est structuré en deux parties. La première partie couvre les concepts basiques suivants&nbsp;:
```
Comment (Agathebadia) [inline guides/release/tutorial/part-1/index.md:10]:
```suggestion
Ce tutoriel est structuré en deux parties. La première partie couvre les concepts suivants&nbsp;:
```

Comment (Agathebadia) [reply guides/release/tutorial/part-1/index.md:10]: I would see this as easier to read - I don't think 'basiques' here is that much needed for the comprehension. 'les bases suivantes' or 'les concepts suivants' seem enough.

**Review by Agathebadia — APPROVED** [review]
I added a small suggestion, more of a nit, otherwise, LGTM ✅

File: guides/release/tutorial/part-1/index.md (hunk 1 above)
Comment (MrChocolatine) [reply guides/release/tutorial/part-1/index.md:10]: Agree

```
Ce tutoriel est structuré en deux parties. La première partie couvre les concepts suivants :
```

File: guides/release/tutorial/part-1/index.md [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
+- Tests automatisés
+- Travailler avec un serveur de données
 
-- Using Ember CLI
-- Navigating the file and folder structure of an Ember app
-- Building and linking between pages
-- Templates and components
-- Automated testing
-- Working with server data
+La seconde partie du tutoriel s'appuie sur ces concepts pour passer l'apprentissage au niveau suivant.
```
Comment (MrChocolatine) [inline guides/release/tutorial/part-1/index.md:19]: "pour passer l'apprentissage au niveau suivant." feels to much word-for-word.

What about:

```suggestion
La seconde partie du tutoriel s'appuie sur ces concepts pour monter en compétences.
```

**Review by MrChocolatine — APPROVED** [review]

**Pull request merged**